dubbo-metrics/dubbo-metrics-api/src/main/java/org/apache/dubbo/metrics/model/MetricsKey.java

```diff
@@ -47,6 +47,9 @@ public enum MetricsKey {
     METRIC_RT_P99("dubbo.%s.rt.seconds.p99", "Response Time P99"),
     METRIC_RT_P95("dubbo.%s.rt.seconds.p95", "Response Time P95"),
 
+    PROVIDER_METRIC_RT_HISTOGRAM("dubbo.%s.rt.seconds.histogram", "Response Time Histogram"),
```

Comment:
please change the name  of PROVIDEER_METRIC_RT_HISTORYGRAM to   
METRIC_RT_HISTORYGRAM

seconds may be error , need change to  milliseconds

Reply:
modified